bin/reports/compile_member_counts_report.rb

```diff
@@ -14,7 +14,7 @@ def make_outf(output_dir)
   else
     FileUtils.mkdir_p(output_dir)
     ymd = Time.new.strftime("%F")
-    File.join(output_dir, "member_counts_#{ymd}.tsv")
+    fn = File.join(output_dir, "member_counts_#{ymd}.tsv")
```

Comment:
Any reason why not `File.open(File.join(output_dir, "member_counts_#{ymd}.tsv"), "w")`?

Reply:
That's usually what the first iteration of my code looks like. 
I prefer more short & simple lines, where you can almost tell what it does without reading the whole thing, just by seeing the shape of it.